script_library/pre-flight-checks.sh

```diff
@@ -106,7 +106,7 @@ check_ansible_requirements (){
     fi
     # We need ansible version 2.8 minimum
     if [[ $(ansible --version | awk 'NR==1 { gsub(/[.]/,""); print substr($2,0,2); }' ) -lt "28" ]]; then
```

Comment:
We should change the actual check if it's true that 2.8.3 is what is required. Do you know why 2.8.1 doesn't work?